src/game/common/system/gamememory.cpp

```diff
@@ -41,12 +41,27 @@ void *New_New(size_t bytes)
     return g_dynamicMemoryAllocator->Allocate_Bytes(bytes);
 }
 
+void *New_New_Dbg(size_t bytes, char const *file, int line)
+{
+    ++(Make_Global<int>(PICK_ADDRESS(0x00A29B9C, 0x00E1B368)));
```

Comment:
May I suggest to separate these into
```c++
auto var_name = (Make_Global<int>(PICK_ADDRESS(0x00A29B9C, 0x00E1B368)))
++var_name;
```
This way it will be possible to inspect variable easily in debugger.